client/mixins/itemHelper.ts

```diff
@@ -14,6 +18,10 @@ declare module '@nuxt/types' {
     canMarkWatched: (item: BaseItemDto) => boolean;
     getItemDetailsLink: (item: BaseItemDto, overrideType?: string) => string;
     getItemIcon: (item: BaseItemDto | BaseItemPerson) => string;
+    getMediaStreams: (
```

Comment:
Why take a stream type as a string, instead of making different filtering methods?

The latter is cleaner, since it doesn't rely on strings to work, and more explicit as to what it does.

Kind of similar to the Lodash shorthand thing we discussed the other day on Matrix :smile:

Reply:
Wouldn't that include a lot of redundancy in the code though?